src/main/java/com/endava/soa_5/basic/Exercise_1.java

```diff
@@ -0,0 +1,36 @@
+package com.endava.soa_5.basic;
+
+public class Exercise_1 {
+    //Write a method that converts a total number of seconds to hours, minutes, and seconds.
+
+    int totalSec, sec, min, hour;
+
+    Exercise_1() {
+    }
+
+    Exercise_1(int totalSec) {
+        this.totalSec = totalSec;
+    }
+
+    int calculateSec() {
+        sec = totalSec % 60;
+        return sec;
+    }
+
+    int calculateMin() {
+        min = (totalSec - sec) / 60 % 60;
```

Comment:
Here and in the next methods might be a slight issue. It seems that all these methods depend on each other, for example when you create the class, you could start by running calculateHour, then calculateSec, then run messageForResult and you will get 0 min and maybe some other strange data. If you run calculateHour before the other 2 above it, the values for sec and min would be 0, which might mess up the calculations.
You could think of a solution how to avoid these kind of issues here

Reply:
Done!